conf/alice.config

```diff
@@ -0,0 +1,39 @@
+params {
+      config_profile_name = 'ALICE'
+      config_profile_description = 'Profile for use on Academic Leiden Interdisciplinary Cluster Environment (ALICE).'
+      config_profile_contact = 'Bjorn Peare Bartholdy (@osteobjorn)'
+      config_profile_url = 'https://wiki.alice.universiteitleiden.nl/index.php?title=ALICE_User_Documentation_Wiki'
+      max_cpus = 24
+      max_memory = 240.GB
+      max_time = 168.h
+}
+
+process {
+  executor = 'slurm'
+  queue = { task.time < 3.h ? 'cpu-short' : task.time < 24.h ? 'cpu-medium' : 'cpu-long' }
+}
+
+singularity {
+  enabled = true
+  autoMounts = true
+}
```

Comment:
No cache directory specified for Singularity?

Reply:
As far as I'm aware there's no shared storage on the cluster. The 'shared data' are stored under each user's username and sym-linked to the home directory

Reply:
Often doesn't work for large multi-group clusters, same at a lot of German HPCs. Have to set it per person